XML/TestCases/BenchmarkTests-MPI.xml

```diff
@@ -0,0 +1,29 @@
+<TestCases>
+<!-- MVAPICH MPI -->
+    <test>
+		<testName>INFINIBAND-MVAPICH-MPI-OMB-2VM</testName>
```

Comment:
Can you check here for indentation? I am not sure if you have double tabs or Github UI issue.